templates/long_range_colloquium.md.j2

```diff
@@ -29,7 +29,7 @@ Attendance is open to everyone, but please register to attend! If you would like
 {% if talk.registration_url %}
 If the form below doesn't work, [register directly]({{ talk.registration_url }})
 
-<iframe width="100%" height="940" src="{{ talk.registration_url }}" frameborder="0" marginheight="0" marginwidth="0" style="overflow-x:hidden"></iframe>
+<iframe width="100%" height="940" src="{{ talk.registration_form }}" frameborder="0" marginheight="0" marginwidth="0" style="overflow-x:hidden"></iframe>
```

Comment:
Will this work? The source of `talk.registration_form` has quotation marks, doesn't this break the HTML?